source/grid/tria.cc

```diff
@@ -1064,8 +1064,8 @@ namespace internal
     else // if (mpisize > 1)
 #endif
       {
-        (void)global_first_cell;
-        (void)global_num_cells;
+        // (void)global_first_cell;
+        // (void)global_num_cells;
         (void)mpi_communicator;
```

Comment:
Can you verify if that statement is still necessary ? :)

Reply:
Are you referring to line 1069?

Reply:
Yes, the   (void)mpi_communicator; line.

Reply:
If that is the case, I didn't remove it because I only did the first 5, another person did line 1069 I think :)

Reply:
Perfect :). My bad sorry